### ISSUE TITLE
Turkish Locale Bug

### ISSUE THREAD

**Issue by webfolderio** [Forge-Srl/jjbridge-engine-v8#1]
Hi,

The below code snippet prints incorrect result.

```java
import jjbridge.api.JSEngine;
import jjbridge.api.runtime.JSReference;
import jjbridge.api.runtime.JSRuntime;
import jjbridge.api.value.JSString;
import jjbridge.engine.v8.V8Engine;

public class Main {

	public static void main(String[] args) throws Exception {
		JSEngine engine = new V8Engine();
		try (JSRuntime runtime = engine.newRuntime()) {
			JSReference executeScript = runtime.executeScript("'ıIiİ'.toLocaleUpperCase('tr-TR')");
			JSString resolveReference = runtime.resolveReference(executeScript);
			System.out.println(resolveReference.getValue());
		} catch (RuntimeException e) {
			e.printStackTrace();
		}
	}
}
```

Result (Wrong): `IIIİ`

Correct Result: `IIİİ`

Ref.: https://blog.codinghorror.com/whats-wrong-with-turkey/

**Comment by tbossi** [comment]
Hi, thanks for reaching out!

jjbridge does not directly handle i18n, meaning it leverages V8 for that. Anyway just to be sure I made a quick check and can confirm the problem is in the V8 engine.
In fact even this fails:
`runtime.executeScript("'ıIiİ'.toLocaleUpperCase('tr-TR') === 'IIİİ'")`
it should be `true` but resolves to `false` instead.

Now, I guess the issue is in the specific V8 binary we used to build jjbridge; I know V8 has some compile-time flags and settings regarding i18n but I need more time to investigate the problem.
